src/main/python/camdkit/model.py

```diff
@@ -52,55 +53,669 @@ class LensSerialNumber(StringParameter):
   canonical_name = "lensSerialNumber"
   sampling = Sampling.STATIC
   units = None
+  section = "lens"
```

Comment:
Including _lens_ in the name and having a section called _lens_ is redundant.

We should decide to group parameters by name, explicit section, or tags.

Reply:
This is complete in latest commits